pkg/pool-manager/pool.go

```diff
@@ -72,16 +77,27 @@ func NewPoolManager(kubeClient kubernetes.Interface, rangeStart, rangeEnd net.Ha
 	currentMac := make(net.HardwareAddr, len(rangeStart))
 	copy(currentMac, rangeStart)
 
+	podOptInValue, err := getOptInLabelValueFromMutatingWebhookConfiguration(kubeClient, mutatingWebhookConfigName, "mutatepods.kubemacpool.io", names.NAMESPACE_OPT_IN_LABEL_PODS)
+	if err != nil {
+		return nil, fmt.Errorf("Failed to get Namespace selector pod label value from MutatingWebhookConfiguration %v", err)
```

Comment:
`errors.Wrap(err, "failed to get Namespace selector pod label value from MutatingWebhookConfiguration"`